app/templates/_instructions.html

```diff
@@ -0,0 +1,174 @@
+      <span class="kr">const</span> <span class="nx">response</span> <span class="o">=</span> <span class="nx">await</span> <span class="nx">fetch</span><span class="p">(</span><span class="nx">resourceUrl</span><span class="p">)</span>
+      <span class="nx">resource</span><span class="p">.</span><span class="nx">descriptor</span><span class="p">.</span><span class="nx">_values</span> <span class="o">=</span> <span class="nx">await</span> <span class="nx">response</span><span class="p">.</span><span class="nx">json</span><span class="p">()</span>
+    <span class="p">}</span> <span class="k">else</span> <span class="p">{</span>
+      <span class="c1">// we assume resource is tabular for now ...</span>
+      <span class="kr">const</span> <span class="nx">table</span> <span class="o">=</span> <span class="nx">await</span> <span class="nx">resource</span><span class="p">.</span><span class="nx">table</span>
+      <span class="c1">// rows are simple arrays -- we can convert to objects elsewhere as needed</span>
+      <span class="kr">const</span> <span class="nx">rowsAsObjects</span> <span class="o">=</span> <span class="kc">false</span>
+      <span class="nx">resource</span><span class="p">.</span><span class="nx">descriptor</span><span class="p">.</span><span class="nx">_values</span> <span class="o">=</span> <span class="nx">await</span> <span class="nx">table</span><span class="p">.</span><span class="nx">read</span><span class="p">(</span><span class="nx">rowsAsObjects</span><span class="p">)</span>
+    <span class="p">}</span>
+  <span class="p">}))</span>
+
+  <span class="c1">// see the data package object</span>
+  <span class="nx">console</span><span class="p">.</span><span class="nx">dir</span><span class="p">(</span><span class="nx">dp</span><span class="p">)</span>
+
+  <span class="c1">// data itself is stored in Resource object, e.g. to access first resource:</span>
+  <span class="nx">console</span><span class="p">.</span><span class="nx">log</span><span class="p">(</span><span class="nx">dp</span><span class="p">.</span><span class="nx">resources</span><span class="p">[</span><span class="mi">0</span><span class="p">].</span><span class="nx">_values</span><span class="p">)</span>
+
+  <span class="k">return</span> <span class="nx">dp</span>
+<span class="p">}</span>
+
+
+<span class="nx">fetchDataPackageAndData</span><span class="p">(</span><span class="s1">'https://bits.datapackaged.com/metadata/{{ dataset.owner }}/{{ dataset.name }}/_v/latest/datapackage.json'</span><span class="p">);</span>
+</pre>
+</div>
+
+<p>Our JavaScript is written using ES6 features. We are using <code>node.js v7.4.0</code> and passing <code>--harmony</code> option to enable ES6:</p>
+<p>
+<div class="highlight">
+<pre>$ node --harmony index.js</pre>
```

Comment:
This is a bit unclear for me should I install or no? If yes why it is at the end of the page?